src/components/tables/EditableTranscriptsTable.tsx

```diff
@@ -116,7 +116,9 @@ const EditableTranscriptsTable = () => {
               setClaimState((prev) => ({ ...prev, rowId: -1 }));
               toast({
                 status: "error",
-                title: "Failed to claim transcript",
+                title: error.message.includes("finish editing")
```

Comment:
can't we pass the error directly from the backend rather than checking for a particular phrase, a single source of error would be better to debug

Reply:
The error message is actually coming from the backend. This is just to set the title dynamically. Since the backend only returns the message and not title

Reply:
The other thing we could do is display the descriptive message in the toast without the title